src/imitation/scripts/train_adversarial.py

```diff
@@ -22,16 +23,26 @@
 logger = logging.getLogger("imitation.scripts.train_adversarial")
 
 
-def save(trainer, save_path):
+def save_checkpoint(
+    trainer: common.AdversarialTrainer,
+    log_dir: str,
+    eval_venv: vec_env.VecEnv,
+    round_str: str,
+) -> None:
     """Save discriminator and generator."""
+    save_path = os.path.join(log_dir, "checkpoints", round_str)
```

Comment:
I have a PR for replacing os.path with pathlib in most places, but might as well keep it consistent for now until that's merged.